<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationWindow</class>
 <widget class="QDialog" name="ConfigurationWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>476</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>476</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>476</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>410</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pteConfiguration">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>381</width>
     <height>351</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Your url: www.vk.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Change yout configurations:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>450</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
